### DISCUSSION TITLE
Интеграция с Yandex Disk

### DISCUSSION THREAD

**Discussion by Alexmeb** [totumonline/totum-ru-issues-and-discussions#252] [Есть идея, а что если...]
Может быть подружим?
Было бы здорово, если бы была возможность кнопкой создавать/удалять/переименовывать папку на яндекс диске))

**Comment by ttm-alexeypolunin** [comment]
https://habr.com/ru/company/yandex/blog/227377/

**Reply by Alexmeb** [reply]
Хах) мы же не программисты....
